input/docs/running-builds/configuration/index.cshtml

```diff
@@ -0,0 +1,9 @@
+Order: 40
+RedirectFrom: docs/running-builds/configuration
```

Comment:
This redirects to itself and leads to an infinite redirect

Reply:
Yes, that does not seem right. I'll fix it.